runner-plugins/runner-scribe/main.go

```diff
@@ -28,8 +28,9 @@ import (
 // URL and Source are mandatory settings
 type config struct {
 	MozDef struct {
-		URL    string // URL to post events to MozDef
-		Source string // Source identifier for vulnerability events
+		URL      string // URL to post events to MozDef
+		Source   string // Source identifier for vulnerability events
+		UseProxy bool
```

Comment:
Consider adding a `// comment` line or removing the two above.

Reply:
Fair suggestion! Done.